contrib/cni/README.md

```diff
@@ -40,19 +45,19 @@ two plugins necessary for the example CNI configurations are `loopback` and
 
 #### CNI Plugin Installation From Source
 
-This tutorial will use the latest version of `CNI` plugins from the master branch and build it from source.
+This tutorial will use the latest version of `CNI` and build it from source.
```

Comment:
I think it's still worth mentioning `CNI` plugins.

Maybe reword this to
This tutorial will use the latest version of the `CNI` plugins and build it from source.

Reply:
ehr yeah, fatfingered the removal.